### PR TITLE
RAC-6011: Refactor continuous test pipeline

### PR DESCRIPTION
**Pull request by PengTian0**
Add pipeline: Continuous Function Test and Continuous Unit Test.
Test job:
http://rackhdci.lss.emc.com/job/Archiving/job/tian/job/FunctionTest/
http://rackhdci.lss.emc.com/job/Archiving/job/tian/job/ContinuousUnitTest/

@panpan0000 @changev @HQebupt  @kellylu2sym

### CODE REVIEW

**Comment by PengTian0** [comment]
test this please

**Review by panpan0000 — APPROVED** [review]

File: src/pipeline/common/Manifest.groovy [hunk 1]
```diff
@@ -1,274 +1,272 @@
...
+//
+// Inputs:
+// 1. url : the URL to be downloaded, typically from Bintray
+// 2. target_dir: the target folder to be downloaded( NOTE: it will always be a folder instead of a file! ), if folder not exist, wget will create for you if appliable.
+//
+// Outputs:
+// the path (/path/to/manifest_fname) of the downloaded file
+//
+//=============================================================
+def downloadManifest(String url, String target_dir){
```
Comment (changev) [inline src/pipeline/common/Manifest.groovy:190]: I'd prefer this function to support user/pass

**Review by iceiilin — APPROVED** [review]

File: src/pipeline/common/Manifest.groovy [hunk 2]
```diff
@@ -1,274 +1,272 @@
...
+// stash a manifest file: to save the file(s) in Jenkins server for later usage from other vmslave
+//
+// Inputs: 
+// stash_name   : the "key" of the locker
+// manifest_path: where the manifest file is located (path/file)
+//
+// Output:
+// a Groovy Map, in which the "stash_name" is the key, and "stash_path" is the path (useful to find the files after unstash)
+// ==================================================================
+def stashManifest(String stash_name, String manifest_path){
```
Comment (changev) [inline src/pipeline/common/Manifest.groovy:207]: Is stashManifest different from a common stashFile?

Comment (panpan0000) [reply src/pipeline/common/Manifest.groovy:207]: actually. not .

**Pull request merged**